portal-api/api-tests/v1/utilisation-report-service/due-reports.api-test.js

```diff
@@ -0,0 +1,94 @@
+const wipeDB = require('../../wipeDB');
+const app = require('../../../src/createApp');
+const { as, get } = require('../../api')(app);
+const testUserCache = require('../../api-test-users');
+const { withClientAuthenticationTests } = require('../../common-tests/client-authentication-tests');
+const { withRoleAuthorisationTests } = require('../../common-tests/role-authorisation-tests');
+const { PAYMENT_REPORT_OFFICER } = require('../../../src/v1/roles/roles');
+const { DB_COLLECTIONS } = require('../../fixtures/constants');
+const { insertOneUtilisationReportDetails } = require('../../insertUtilisationReportDetails');
+
+describe('GET /v1/due-reports/:bankId', () => {
+  const dueReportsUrl = (bankId) => `/v1/due-reports/${bankId}`;
+  let aPaymentReportOfficer;
+  let mockUtilisationReport;
+  let testUsers;
+  let matchingBankId;
+
+  const expectedDueReports = [{
+    month: 'December',
+    year: 2022,
+  }, {
+    month: 'January',
+    year: 2023,
+  }, {
+    month: 'February',
+    year: 2023,
+  }];
+
+  beforeAll(async () => {
+    testUsers = await testUserCache.initialise(app);
+    aPaymentReportOfficer = testUsers().withRole(PAYMENT_REPORT_OFFICER).one();
+    matchingBankId = aPaymentReportOfficer.bank.id;
+
+    const mockDate = new Date(2023, 2, 1); // Mar 1st 2023
```

Comment:
This can also be written like this which can be useful to avoid the zero index confusion
```suggestion
    const mockDate = new Date('2023-03-01'); // Mar 1st 2023
```